src/app/dashboard/mortgages/_forms/customer-info-form.tsx

```diff
@@ -120,73 +121,127 @@ interface Props {
-        <PhoneNumberInputElement name="contact" label="Contact No." />
-        <SelectElement name="education" label="Education" options={educationOptions} />
-        <SelectElement name="marital_status" label="Marital Status" options={maritalStatusOptions} />
-        <InputElement name="favorite_city" label="Favorite City" />
-        <InputElement name="number_of_family_member_in_uae" label="Number of Family Members in UAE" type="number" />
-        <InputElement name="years_in_uae" label="Years in UAE" />
-        <InputElement name="annual_rental_income" label="Annual Rental Income" />
-        <InputElement name="uae_residence_address" label="UAE Residence Address" />
-        <InputElement name="home_country_address" label="Home Country Address" />
-
-        <h2 className="pt-5 text-xl font-bold ">HOME COUNTRY REFERENCES</h2>
-
-        <InputElement name="home_country_reference_1.name" label="Name" />
-        <SelectElement
-          name="home_country_reference_1.relationship"
-          label="Relationship"
-          options={relationshipOptions}
-        />
-        <PhoneNumberInputElement name="home_country_reference_1.mobile" label="Mobile" />
-
-        <div className="mx-auto h-[3px] w-1/2 rounded-full bg-black/20" />
-
-        <InputElement name="home_country_reference_2.name" label="Name" />
-        <SelectElement
-          name="home_country_reference_2.relationship"
-          label="Relationship"
-          options={relationshipOptions}
-        />
-        <PhoneNumberInputElement name="home_country_reference_2.mobile" label="Mobile" />
-
-        <h2 className="pt-5 text-xl font-bold">PERSONAL REFERENCES IN UAE</h2>
-        <InputElement name="uae_reference_1.name" label="Name" />
-        <SelectElement name="uae_reference_1.relationship" label="Relationship" options={relationshipOptions} />
-        <PhoneNumberInputElement name="uae_reference_1.mobile" label="Mobile" />
-
-        <div className="mx-auto h-[3px] w-1/2 rounded-full bg-black/20" />
-
-        <InputElement name="uae_reference_2.name" label="Name" />
-        <SelectElement name="uae_reference_2.relationship" label="Relationship" options={relationshipOptions} />
-        <PhoneNumberInputElement name="uae_reference_2.mobile" label="Mobile" />
-
-        <Button type="submit" className="w-full">
-          Next
-        </Button>
-        <BackButton route={`${PageRoutes.dashboard.MORTGAGES}/${mortgageId}/transaction-info`} />
-      </form>
-    </Form>
-  )
+    const storedValue = localStorage.getItem('mortgage/customer-info')
+    const defaultValues: z.infer<typeof formSchema> = storedValue !== null && JSON.parse(storedValue)
+
+    const form = useForm<z.infer<typeof formSchema>>({
+        resolver: zodResolver(formSchema),
+        defaultValues
+    })
+
+    function onSubmit(values: z.infer<typeof formSchema>) {
+        console.log({ values })
+        onSave("mortgage/customer-info", values)
```

Comment:
localstorage constants

Reply:
have to create new constants for these because these forms are used with dynamic route, thats why i kept them as string only for now